etc/profile-a-l/keepassxc.profile

```diff
@@ -62,6 +62,7 @@ include disable-xdg.inc
 #whitelist ${HOME}/.config/KeePassXCrc
 #include whitelist-common.inc
 
+whitelist ${RUNUSER}/wayland-*
```

Comment:
Actually this should be `whitelist ${RUNUSER}/$WAYLAND_DISPLAY`. Are environment variables usable here?

Reply:
`$WAYLAND_DISPLAY` can be `/tmp/tomatoes/landway`.

Reply:
> To find the Unix socket to connect to, most implementations just do what libwayland does:
> 
>     If WAYLAND_SOCKET is set, interpret it as a file descriptor number on which the connection is already established, assuming that the parent process configured the connection for us.
>     If WAYLAND_DISPLAY is set, concat with XDG_RUNTIME_DIR to form the path to the Unix socket.
>     Assume the socket name is wayland-0 and concat with XDG_RUNTIME_DIR to form the path to the Unix socket.
>     Give up.

https://wayland-book.com/protocol-design/wire-protocol.html

I don't think it can be an absolute path.

Reply:
If you have suggestions on how to support _any_ value of `WAYLAND_DISPLAY`, I can update the PR, but this works for the default configuration on any compositor.

Reply:
> If you have suggestions on how to support any value of WAYLAND_DISPLAY, I can update the PR, but this works for the default configuration on any compositor.

You would have to add a `${WAYLAND_DISPLAY}` macro.

---

https://wayland.freedesktop.org/docs/html/ch04.html#sect-Protocol-Wire-Format

> Beginning in Wayland 1.15, implementations can optionally support server socket endpoints located at arbitrary locations in the filesystem by setting WAYLAND_DISPLAY to the absolute path at which the server endpoint listens.